src/_macosx.m

```diff
@@ -47,6 +47,7 @@
 #ifndef COMPILING_FOR_10_5
 static int ngc = 0;    /* The number of graphics contexts in use */
 
+#include <Carbon/Carbon.h>
```

Comment:
Does this put a min OSX version constraint in?  I don't follow mac at all, but have a vague notion that carbon is 'new'.